timer/__init__.py

```diff
@@ -35,11 +41,13 @@ def timeout():
             subprocess.Popen(["aplay", soundPath])
             global timers
             timers.remove(self)
-            subtext="Timed out at %s" % strftime("%X", localtime(self.end))
-            if self.name:
-                subprocess.Popen(['notify-send', 'Timer "%s"' % self.name, '-t', '0', subtext])
-            else:
-                subprocess.Popen(['notify-send', 'Timer', '-t', '0', subtext])
+
+            title = 'Timer "%s"' % self.name if self.name else 'Timer'
+            text = "Timed out at %s" % strftime("%X", localtime(self.end))
+
+            bus = dbus.SessionBus()
+            notify = dbus.Interface(bus.get_object(dbusItem, dbusPath), dbusInterface)
+            notify.Notify(__prettyname__, random.randint(0, 100000), iconPath, title, text, '', '', 0)
```

Comment:
I see no point in this random. If possible use None otherwise send 0. No need to draw a random REPLACE ID.